pom.xml

```diff
@@ -138,6 +138,11 @@
             <artifactId>configuration-as-code</artifactId>
             <version>0.8-alpha</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>JiraTestResultReporter</artifactId>
+            <version>2.0.6</version>
```

Comment:
It would be great to make this dependency optional at least (or scoped to `test`). This plugin is not widely used, and it's bad to enforce it's wide installation IMHO.

Once the plugin has extension points for steps, the integration can be converted to the optional extension

Reply:
Oh! is left by mistake. I will remove it.